.github/workflows/build-fe-dev.yml

```
@@ -16,14 +16,29 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: move to frontend path
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18.x'
+          cache: 'yarn'
+          cache-dependency-path: ${{ vars.FE_DIRECTORY }}/yarn.lock
+
+      - name: Install dependencies
         run: |
-          cd ./frontend
+          yarn install
```

```suggestion
          yarn install --frozen-lockfile
```
으로 수정부탁드립니다🙂

`yarn install` 은 `package.json`에 명시된 라이브러리의 버전 범위를 보고 사용 가능한 최신 버전을 찾아서 설치합니다. 그 결과를 yarn.lock에 기록하기 때문에 `yarn.lock`이 수정될 수 있고 또한 항상 동일한 버전의 라이브러리가 설치된다는 보장이 없습니다.

`yarn install --frozen-lockfile`은 `yarn.lock` 파일을 보고 딱 정해진 버전의 라이브러리를 설치하기 때문에 해당 문제를 해결할 수 있습니다.

[공식문서 참고](https://classic.yarnpkg.com/lang/en/docs/cli/install/#toc-yarn-install)

---

감사합니다 반영 완료했습니다~